spec/sidekiq-instrument/client_middleware_spec.rb

```diff
@@ -2,6 +2,10 @@
 
 RSpec.describe Sidekiq::Instrument::ClientMiddleware do
   describe '#call' do
+    let(:worker_metric_name) do
+      "sidekiq_instrument_trace_workers::in_queue"
```

Comment:
Nit: This is the only place we're using double quotes for some reason; can we keep it consistent with the rest of the file and use single quotes?
```suggestion
      'sidekiq_instrument_trace_workers::in_queue'
```

Reply:
I'm also having a hard time wrapping my head around the use of `in_queue`, `inqueue`, and `enqueue` 😅 Example uses: [in_queue](https://github.com/enova/sidekiq-instrument/blob/0206e28a5fd3aa5508e8cc9d6dc4ed5617817a42/spec/sidekiq-instrument/client_middleware_spec.rb#L44), [inqueue](https://github.com/enova/sidekiq-instrument/blob/0206e28a5fd3aa5508e8cc9d6dc4ed5617817a42/lib/sidekiq/instrument/worker.rb#L67C1-L68), [enqueue](https://github.com/enova/sidekiq-instrument/blob/0206e28a5fd3aa5508e8cc9d6dc4ed5617817a42/lib/sidekiq/instrument/middleware/client.rb#L15C1-L16).

Is it too late to consolidate or better distinguish these metric names?